### PR TITLE
Remove AMQP 1.0 protocol support

### PR DESCRIPTION
**Pull request by ppatierno**
As per the corresponding [proposal](https://github.com/strimzi/proposals/blob/main/042-remove-bridge-amqp-support.md), this PR is about removing the AMQP 1.0 protocol support in the bridge.
Right now:

* it removes all the `amqp` related packages with source code and tests.
* it removes all the AMQP 1.0 related design documentation and Qpid Dispatch Router configuration for testing.
* it changes the main part of the bridge where the AMQP 1.0 support was involved in order to remove its start up and usage.
* it updates the configuration properties file to remove AMQP 1.0 related parameters and Docker image script to run the bridge without AMQP 1.0 support

This is a first PR and I would like to open more because the source code can be now simplified but at same time avoiding a huge single PR that is difficult to review.

### CODE REVIEW

File: development-docs/TESTING.md [hunk 1]
```diff
@@ -34,10 +34,7 @@ You can find tests inside `src.test.java` package. Moreover we have the auxiliar
 - `Clients` clients for testing overall communication of the `Kafka Bridge`
 - `Facades` encapsulation of the standalone `Kafka` and `AdminClient` instance
 
-Furthermore we divide our test suite on two separate parts:
-
-1. `Http`package, where you can find all test cases related to the `Http Kafka Bridge`
-2. `Amqp`package, where you can find all tests cases related to the `Amqp Kafka Bridge`
+The test suite has provides a `Http`package, where you can find all test cases related to the `Http Kafka Bridge`.
```
Comment (scholzj) [inline development-docs/TESTING.md:37]: Is the package really `Http`? Or is it `http`? Package names are normally lowercase.

```suggestion
The test suite has provides a `Http` package, where you can find all test cases related to the `Http Kafka Bridge`.
```

File: development-docs/TESTING.md [hunk 2]
```diff
@@ -77,8 +73,7 @@ to add the second option. Rest of these approaches are not so essential.
 ## Test Phases
 
 In general, we use classic test phases: `setup`, `exercise`, `verify` and `teardown`. Every phase will be described for the
-`src.test.java.io.strimzi.kafka.bridge.http` package. It is very similar with `src.test.java.io.strimzi.kafka.bridge.amqp` only
-one difference is that `http` package using Strimzi `Kafka` container by `test-containers` and `amqp` package using KafkaFacade.
+`src.test.java.io.strimzi.kafka.bridge.http` package.
```
Comment (scholzj) [inline development-docs/TESTING.md:76]: I guess the actual package is `io.strimzi.kafka.bridge.http`?

**Review by scholzj — COMMENTED** [review]
I left some nits ... mostly in the docs part such as some typos etc.

File: development-docs/TESTING.md (hunk 2 above)
Comment (ppatierno) [reply development-docs/TESTING.md]: Yes you are right, I think the intent of who wrote this testing doc was about highlighting the full path under the "test" but as you said the package name is what you wrote. I rephrased a little bit.

**Review by scholzj — APPROVED** [review]

**Pull request merged**